best_practices/business-logic.rst

```diff
@@ -57,8 +57,9 @@ post URL. Let's create a new ``Slugger`` class inside ``src/Utils/``::
     }
 
 If you're using the :ref:`default services.yaml configuration <service-container-services-load-example>`,
-this class is auto-registered as a service whose ID is ``App\Utils\Slugger`` (or
-simply ``Slugger::class`` if the class is already imported in your code).
+this class is auto-registered as a service whose ID is ``App\Utils\Slugger`` (as
```

Comment:
s/whose ID is/with the ID/

afaik "who" only applies to people, not to things.